sycl/include/CL/sycl/detail/image_impl.hpp

```diff
@@ -245,15 +247,19 @@ class __SYCL_EXPORT image_impl final : public SYCLMemObjT {
   RT::PiMemImageDesc getImageDesc(bool InitFromHostPtr) {
     RT::PiMemImageDesc Desc;
     Desc.image_type = getImageType();
-    Desc.image_width = MRange[0];
-    Desc.image_height = Dimensions > 1 ? MRange[1] : 1;
-    Desc.image_depth = Dimensions > 2 ? MRange[2] : 1;
+
+    // MRange<> is [width], [width,height], or [width,height,depth] (which
+    // is different than MAccessRange, etc in bufffers)
+    static constexpr int x_term_pos = 0, y_term_pos = 1, z_term_pos = 2;
+    Desc.image_width = MRange[x_term_pos];
```

Comment:
Could you please clarify what modifications in this file do?

Reply:
These do nothing, but make it clearer what is happening.   I added the comment and var names to make things explicit.  It may not seem like much, but since buffer accessors and image accessors use different argument ordering, it seemed important to be very clear.